packages/common/src/store/user-list/related-artists/reducers.ts

```diff
@@ -0,0 +1,35 @@
+import { combineReducers } from 'redux'
+import { createReducer, ActionType } from 'typesafe-actions'
+
+import { UserListReducerFactory } from '../reducer'
+
+import * as actions from './actions'
+import { RelatedArtistsOwnState, RELATED_ARTISTS_USER_LIST_TAG } from './types'
+
+type RelatedArtistsActions = ActionType<typeof actions>
+
+const userListReducer = UserListReducerFactory.createReducer({
+  tag: RELATED_ARTISTS_USER_LIST_TAG,
+  pageSize: 15
+})
+
+const initialState = {
+  id: null
+}
+
+const relatedArtistsPageReducer = createReducer<
```

Comment:
Is there a reason we aren't using `createSlice` here? For consistency with the other user-lists?

Reply:
Yeah every other user-list used createReducer. I'm sure it'd work either way though. Thoughts?

Reply:
I would prefer createSlice bc the other ones will be migrated eventually (hopefully)

Reply:
resolved to stick with the implementaiton consistent with the parallel user-list slices, but filing a work item to refactor the whole thing